src/arrayMethodSort.js

```diff
@@ -3,9 +3,37 @@
 /**
  * Implement method Sort
  */
+function stringSortFunction(a, b) {
+  if (String(a) < String(b)) {
+    return -1;
+  } else if (String(a) > String(b)) {
+    return 1;
+  } else {
+    return 0;
+  }
```

Comment:
```suggestion
  if (String(a) < String(b)) {
    return -1;
  }
  
  if (String(a) > String(b)) {
    return 1;
  }
  
  return 0;
  }
```
No need for if...else here.
Your code will not go further after return anyway.